stix2elevator/convert_stix.py

```diff
@@ -1128,6 +1145,7 @@ def convert_test_mechanism(indicator, indicator_instance):
                         indicator_instance["pattern"] = ", ".join(list_of_strings)
                         indicator_instance["pattern_type"] = "snort"
                     elif isinstance(tm, OpenIOCTestMechanism):
+                        warn("IOC indicator in %s cannot be converted to a STIX pattern", 410, indicator_instance["id"])
```

Comment:
Consistency Question:

Is there a particular reason we have a warn() message only for `OpenIOCTestMechanism`? Neither Snort or Yara can be converted to a STIX Pattern (AFAIK). Should we have it for all instance types?

Reply:
You will only be printed for 2.1.  In 2.1, snort and yara are "allowed" - and are used "as is".  IOC indicators are an extension of STIX 1.x, so a user might expect the elevator to convert them in some way.